backend/services/wallet/tests/functional/test_routes.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from src import app
+
+@pytest.fixture(scope='module')
+def test_client():
+    test_client = app.test_client()
+
+    ctx = app.app_context()
+    ctx.push()
+
+    yield test_client
+
+    ctx.pop()
+
+def test_home_page(test_client):
```

Comment:
Will need new tests for the new endpoints described in the previous comment.

Reply:
done